includes/class-woocommerce.php

```diff
@@ -801,6 +801,10 @@ public function wpdb_table_fix() {
 	public function activated_plugin( $filename ) {
 		include_once dirname( __FILE__ ) . '/admin/helper/class-wc-helper.php';
 
+		if ( 'woocommerce/woocommerce.php' === $filename ) {
```

Comment:
Depending on the installing it maybe never return true, since it could be `woocommerce-master/woocommerce.php` or something else.

Reply:
Good catch, thank you for the suggestion!